### DISCUSSION TITLE
Site not creating

### DISCUSSION THREAD

**Discussion by leahcimto** [upptime/upptime#292] [Q&A] [answered]
Hello! My upptime site is not being created. When I go to the GitHub Pages URL I connected (forumuptime.leahcimto.com) it just displays the README.md. The repository is here: https://github.com/leahcimto/forums-uptime Any help would be appreciated!

**Comment by Hans5958** [comment, marked as answer]
Switch your GitHub Pages branch from `master` to `gh-pages`. Hope that helps.
